src/js/types/metadata.ts

```diff
@@ -0,0 +1,36 @@
+interface DiscoveryOverview {}
+interface DiscoverySearch {}
+interface DiscoveryFields {}
+interface DiscoveryRules {}
```

Comment:
what's the plan for these types? I though there were already types for this stuff?